pcbdraw/plot.py

```diff
@@ -24,7 +27,8 @@
 T = TypeVar("T")
 Numeric = Union[int, float]
 Point = Tuple[Numeric, Numeric]
-Matrix = np.typing.NDArray[np.float32]
+# Matrix = np.typing.NDArray[np.float32] Disabled because we not always have np.typing
+Matrix = Any
```

Comment:
I am not particularly happy about aliasing Matrix to Any as it masks potential type errors. I wonder if there is another way to allow us to not require the typing module but preserve typechecking. 

Maybe define `np.typing.NDarray` and `np.float32` in the catch block of the error after import?

Reply:
Hi @yaqwsx !
I don't really know how to define them. 

As I understand the NDarray is a vector of the class `np.ndarray` with elements that can be `Any` or vectors of the class `np.dtype` containing scalar elements: `NDArray = np.ndarray[Any, np.dtype[ScalarType]]`

I don't know if something like `NDArray = list[Any, list[Any]]` will really help.

`np.float32` seems to be defined as the C float32_t type, in C, and exported to Python. Again, don't know what will really help here. I don't use Python typing and when I tried the `mypy` version available in my system it complained about a lot of PcbDraw stuff.